drivers/interrupt_controller/Kconfig.gic

```diff
@@ -9,21 +9,21 @@ config GIC
 	bool
 
 config GIC_V1
-	bool
+	bool "Enable GICv1 support"
```

Comment:
based on the commit message, I still don't understand why prompt is needed here ?

This should not be configurable by the user, but something the board decides, from how I understand the commit message.

When you create: `zenvm` board, you may create the defconfig with:
`boards/arm/zenvm/Kconfig.defconfig
```
 config GIC_V1
	default y
```
and it will be enabled even for just the `zenvm` board.
This allows the board to select this setting, but not the user.